go/vt/vttablet/tabletserver/schema/tracker.go

```diff
@@ -163,14 +170,72 @@ func (tr *Tracker) process(ctx context.Context) {
 	}
 }
 
+func (tr *Tracker) currentPosition(ctx context.Context) (mysql.Position, error) {
+	conn, err := tr.engine.cp.Connect(ctx)
+	if err != nil {
+		return mysql.Position{}, err
+	}
+	defer conn.Close()
+	return conn.MasterPosition()
+}
+
+func (tr *Tracker) isSchemaVersionTableEmpty(ctx context.Context) (bool, error) {
+	conn, err := tr.engine.GetConnection(ctx)
+	if err != nil {
+		return false, err
+	}
+	defer conn.Recycle()
+	result, err := withDDL.Exec(ctx, "select id from _vt.schema_version limit 1", conn.Exec)
+	if err != nil {
+		return false, err
+	}
+	if result == nil {
+		return false, fmt.Errorf("error querying _vt.schema_version")
+	}
+	if result.RowsAffected == 0 {
```

Comment:
It's better to check `len(result.Rows)`. There is no guarantee that this number will be accurate for non-DMLs. In the future, when we switch to streaming, this will always be 0.